samples/package-lock.json

```diff
@@ -7,32 +7,9 @@
     "azure-devops-node-api": {
```

Comment:
Is this supposed to be changed?

Reply:
Yeah, it should've been before probably. It now matches the package.json